src/main/webapp/i18n/de/course.json

```diff
@@ -132,23 +132,23 @@
             "assessmentDueDate": "Bewertung",
             "cleanup": {
                 "title": "Aufräumen",
-                "question": "Möchtest du den Kurs <strong>{{title}}</strong> wirklich bereinigen? Dadurch werden alle Studenten-Repositories für <strong>Übungen</strong> und <strong>Prüfungen</strong> gelöscht. Diese Aktion kann <strong class='text-danger'>NICHT</strong> rückgängig gemacht werden!"
+                "question": "Möchtest du den Kurs <strong>{{title}}</strong> wirklich bereinigen? Dadurch werden alle Studierenden-Repositories für <strong>Übungen</strong> und <strong>Prüfungen</strong> gelöscht. Diese Aktion kann <strong class='text-danger'>NICHT</strong> rückgängig gemacht werden!"
             },
             "detail": {
                 "title": "Kurs Details:",
                 "assessmentsTitle": "Insgesamte Bewertungen",
                 "complaintsTitle": "Insgesamte Beschwerden",
                 "moreFeedbackTitle": "Insgesamte Feedbackanfragen",
-                "averageStudentScoreTitle": "Durchschnittliche Studentenpunktzahl"
+                "averageStudentScoreTitle": "Durchschnittliche Punktzahl der Studierenden"
             }
         },
         "courseStatistics": {
             "statistics": "Statistiken",
             "course": "Kurs",
             "courseAverage": "Kursdurchschnitt",
             "exerciseAverage": "Aufgabendurchschnitt",
-            "amountOfStudents": "# an Studenten",
-            "activeStudents": "Aktive Studenten"
+            "amountOfStudents": "# Studierende",
+            "activeStudents": "Aktive Studierende"
```

Comment:
Do you need to insert a '#' as well here?

Reply:
as in the english version it's Active Students, I would leave it as such.